src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -1743,12 +1743,9 @@ internal static uint GetRefEscape(BoundExpression expr, uint scopeOfTheContainin
                 case BoundKind.Parameter:
                     var parameter = ((BoundParameter)expr).ParameterSymbol;
 
-                    // byval parameters can escape to method's top level.
-                    // others can be escape further, unless they are ref-like.
-                    // NOTE: "method" here means nearest containing method, lambda or nested method
-                    return parameter.RefKind == RefKind.None || parameter.Type?.IsByRefLikeType == true ?
-                        Binder.TopLevelScope :
-                        Binder.ExternalScope;
+                    // byval parameters can escape to method's top level. Others can be escape further.
+                    // NOTE: "method" here means nearest containing method, lambda or local function.
+                    return parameter.RefKind == RefKind.None ? Binder.TopLevelScope : Binder.ExternalScope;
```

Comment:
So I can create a Span from a by-val argument and return it? Is that intentional? I don't see a test added for this.

Reply:
Never mind, I misread this condition.